samples/cpp/intelligent_scissors.cpp

```diff
@@ -189,7 +189,8 @@ int main( int argc, const char** argv )
     const int EDGE_THRESHOLD_LOW = 50;
     const int EDGE_THRESHOLD_HIGH = 100;
     CommandLineParser parser(argc, argv, keys);
-    parser.about("\nThis program demonstrates implementation of 'intelligent scissors' algorithm\n"
+    parser.about("\nThis program demonstrates implementation of 'intelligent scissors' algorithm:\n"
+                 "http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.138.3811&rep=rep1&type=pdf"
```

Comment:
Please also add paper title and author name:

```
This program demonstrates implementation of 'intelligent scissors' algorithm described by EN Mortensen in paper Intelligent Scissors for Image Composition (http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.138.3811&rep=rep1&type=pdf)
```

And I think we can avoid adding `\n`.  Anyway if length of line less than terminal (80 characters in general) it will be warped.